ansible/roles/openshift_logging/templates/logging-deployer.yaml.j2

```diff
@@ -11,3 +11,4 @@ data:
   kibana-hostname: logs.{{ osalog_clusterid }}.openshift.com
   public-master-url: https://api.{{ osalog_clusterid  }}.openshift.com
   master-url: https://internal.api.{{ osalog_clusterid  }}.openshift.com
+  use-journal: use-journal=true
```

Comment:
This won't force the use of `journald` for the actual containers, this just tells `fluentd` to ONLY collect using `journald`, which means when docker is configured to use `json-file` driver you won't get any logs.